go-controller/pkg/node/openflow_manager.go

```diff
@@ -213,7 +216,16 @@ func (c *openflowManager) updateBridgeFlowCache(subnets []*net.IPNet, extraIPs [
 	// CAUTION: when adding new flows where the in_port is ofPortPatch and the out_port is ofPortPhys, ensure
 	// that dl_src is included in match criteria!
 
-	dftFlows, err := flowsForDefaultBridge(c.defaultBridge, extraIPs)
+	var udnAllowedServicesIPs []net.IP
+	var err error
+	if util.IsNetworkSegmentationSupportEnabled() {
+		udnAllowedServicesIPs, err = c.watchFactory.GetUDNAllowedServicesIPs()
```

Comment:
is updateBridgeFlowCache the right place to be doing this? what if the service changes, we have nodeport watcher that handles services and then updates the openflow cache.

// updateBridgeFlowCache generates the "static" per-bridge flows

I only see updateBridgeFlowCache triggered on:

1. node IP change
2. OFM manager creation
3. node MAC changes
4. UDN Add/Del network

Reply:
I guess in service controller you would need to compare if the sync'ed service matches the one of hte UDNAllowedServiceIPs and then, get the network, and from that you can figure out the mgmt ip. That doesn't seem that complicated since UDN is already integrated into he services controller. Or am I missing something?

Reply:
I'll look into that and push again.

Reply:
Done.